gradle-circle-style/src/test/java/com/palantir/gradle/circlestyle/CircleStyleFinalizerTests.java

```diff
@@ -63,7 +63,8 @@ public void translatesCheckstyleReport() throws IOException, TransformerExceptio
 
         finalizer.createCircleReport();
 
-        String report = Resources.toString(targetFile.toURI().toURL(), StandardCharsets.UTF_8);
+        String report = Resources.toString(targetFile.toURI().toURL(), StandardCharsets.UTF_8)
+                .replaceAll("\\p{Blank}*(?=<)", "");
```

Comment:
indentation of resulting file is different between 11 and 8 where 8 doesn't indent lines and 11 does

Reply:
the root cause is https://bugs.openjdk.java.net/browse/JDK-8087303 and according to the notes it should be possible to set `xml:space` property to match old behaviour